src/main/kotlin/com/github/mitchellolsthoorn/testgenie/settings/TestGenieSettingsComponent.kt

```diff
@@ -12,15 +14,60 @@ import javax.swing.JTextField
  */
 class TestGenieSettingsComponent {
     var panel: JPanel? = null
-    var globalTimeoutTextField = JTextField()
-    var showCoverageCheckBox = JCheckBox("Do you want visualised coverage? ")
+    private var globalTimeoutTextField = JTextField()
```

Comment:
Your user input is not validated. It allows me to input a string where we should expect an Int. In our quick access settings we used JSpinner consider using that.

Reply:
We have removed this field due it not being mentioned by the client.